dist-git-to-src-git/Makefile

```diff
@@ -2,12 +2,14 @@
 
 PACKAGE ?= rpm
 BRANCH ?= c8s
+DIR ?= git.centos.org
```

Comment:
The original idea behind 'make check' was to serve as a very basic test for the script. This is why the dir was called `test` and it is also included in `.gitignore`. 

If we do this, then let's remove `.gitignore`, rename the target to `convert` (as in `make convert`) and update the usage string to avoid future confusion.

Reply:
Ok, PTAL